pkg/export/template.go

```diff
@@ -50,10 +65,16 @@ func parseTemplate(tpl string, colorEnabled bool) (*template.Template, error) {
 	return template.New("").Funcs(templateFuncs).Parse(tpl)
 }
 
-func ExecuteTemplate(w io.Writer, input io.Reader, templateStr string, colorEnabled bool) error {
-	t, err := parseTemplate(templateStr, colorEnabled)
-	if err != nil {
-		return err
+func (t *Template) Execute(input io.Reader, templateStr string) error {
+	w := t.io.Out
+
+	if t.template == nil {
+		template, err := t.parseTemplate(templateStr)
```

Comment:
I like that this avoids parsing the same template over and over in case of pagination. However, wouldn't the parsing be best moved to the NewTemplate function? Then the Execute method wouldn't have to accept `templateStr` via argument anymore, and from the caller's perspective it would be clear that the template is parsed only once.

The way that it stands right now, if I called `Execute()` twice in a row but with different values of `templateStr`, all but the 1st templateStr value would be ignored.